portfolio/src/main/webapp/scripts/generateCircles.js

```diff
@@ -0,0 +1,80 @@
+const GOOGLE_COLORS = ["#f4c20d", "#4885ed", "#3cba54"] // yellow, blue, green
+const ICONS = ["contact", "code", "education", "me", "work", "achievement"];
+
+let contents = [];
+
+const FULL_CIRCLE_DEGREE = 360;
+const SATELLITE_NUMBER = 6;
+const RADIUS = 250;
+const SATELLITE_RADIUS = 90;
+
+const parentdiv = document.getElementById('parentdiv');
+
+const convertCoordinateToString = (coor) => {
```

Comment:
This method name is misleading.  You're not just converting a coordinate to a string, you are also doing math on that coordinate to change its value.

Hypothetical example of why this would matter:  Your display is wrong, and another engineer comes in and tries to figure out why.  They know it must be something modifying the pixels, but as they're skimming your code they see "convertCoordinateToString" and think "ok I don't need to read that function because all it's doing is converting this to a string.  The error must be earlier."

I would suggest putting all the coordinate-related math in one place to make it easy to find/modify later on.

Reply:
Good point! Thanks for bringing this up